src/TLuaInterpreter.cpp

```diff
@@ -2456,7 +2458,7 @@ int TLuaInterpreter::getFont(lua_State* L)
             windowName = QString::fromUtf8(lua_tostring(L, 1));
 
             if (windowName.isEmpty() || windowName.compare(QStringLiteral("main"), Qt::CaseSensitive) == 0) {
-                font = pHost->mDisplayFont.family();
+                font = mudlet::self()->mConsoleMap[pHost]->mUpperPane->fontInfo().family();
```

Comment:
Why is it necessary to reach up into the `mudlet` class to get a pointer to the main `TConsole` when there is the perfectly good one in the `Host` which is already to hand as the `(QPointer<TConsole>) Host::mpConsole` member? - I.e. this line should IMHO be:

    font = pHost->mpConsole->mUpperPane->fontInfo().family();